packages/language/src/pli.langium

```diff
@@ -0,0 +1,662 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+grammar Pl1
+
+entry PliProgram:
+    (statements+=TopLevelStatement)*;
+
+TopLevelStatement: Package | ProcedureStatement | Directives;
```

Comment:
let's discuss these grammar in separate meeting, i have many findings/inputs here